# with more recent versions of Pandas, the imported error was moved to `pandas.errors`
diff -ruN vispr-0.4.14_orig/vispr/results/rna.py vispr-0.4.14/vispr/results/rna.py
--- vispr-0.4.14_orig/vispr/results/rna.py	2023-03-24 13:10:57.148839548 +0000
+++ vispr-0.4.14/vispr/results/rna.py	2023-03-24 13:11:14.032660055 +0000
@@ -11,7 +11,7 @@
 
 from flask import render_template
 import pandas as pd
-from pandas.io.common import EmptyDataError
+from pandas.errors import EmptyDataError
 import numpy as np
 from sklearn.decomposition import PCA
 from scipy.cluster.hierarchy import average, leaves_list, dendrogram
